src/goto-programs/rw_set.cpp

```diff
@@ -36,7 +36,7 @@ void rw_sett::read_write_rec(
     const symbolt *symbol = ns.lookup(symbol_expr.get_identifier());
     if(symbol)
     {
-      if(!symbol->static_lifetime /*&& expr.type().id()=="pointer"*/)
+      if(!symbol->static_lifetime && expr.type().id() != "pointer")
```

Comment:
you can probably use is_pointer_type() here (or `type().is_pointer()`, I don't remember exactly).

Reply:
done